Gruntfile.js

```diff
@@ -94,8 +115,10 @@ module.exports = function(grunt) {
   grunt.loadNpmTasks('grunt-contrib-watch');
   grunt.loadNpmTasks('grunt-symlink');
   grunt.loadNpmTasks('grunt-verbosity');
+  grunt.loadNpmTasks('grunt-postcss');
 
   grunt.registerTask('styleguide', ['run:styleguide', 'verbosity:symlinkquiet']);
+  grunt.registerTask('default', ['postcss:dist']);
```

Comment:
This is duplicating the default task. Remove this line.

You can, instead, move the `'postcss:dist'` tasks in to the styleguide task array in line 120. You will also probably want to ensure that it is ran after calling `grunt sass` but you cannot simply just add this to the end of that command. As you want to make this a dependency I would suggest registering another a new task called `compile` or something that runs both grunt sass and grunt post-css in the same command. 

eg: `grunt.registerTask('compile', ['sass:dist', 'postcss:dist']);`

Reply:
Good catch! And, I really like the `compile` task idea. I've corrected this and have added that task.